kuka_mobile_robot_support/config/differentialMR.rviz

```diff
@@ -0,0 +1,265 @@
+Panels:
```

Comment:
why do you have 3 different rviz configs?
Usually the configs are general enough so that they can be hosted in the `kuka_resources` package, I would appreciate that here too